Documentation/cmdref/cilium-agent.md

```diff
@@ -52,6 +52,7 @@ cilium-agent [flags]
       --disable-endpoint-crd                                  Disable use of CiliumEndpoint CRD
       --disable-k8s-services                                  Disable east-west K8s load balancing by cilium
       --egress-masquerade-interfaces string                   Limit egress masquerading to interface selector
+      --enable-dsr                                            Enable direct server return NodePort BPF (beta)
```

Comment:
Should we do `--node-port-mode=dsr`, so possible values could be {dsr,xdp,legacy} then also used in combination, e.g. `--node-port-mode=dsr,xdp`. Thoughts?

Reply:
SGTM, just maybe `s/legacy/snat`.

Reply:
Yes, `snat` sounds good to me.

Reply:
With `legacy` I actually meant for older kernels that do not have host reachable service support.